cpp/include/copc-lib/copc/extents.hpp

```diff
@@ -0,0 +1,170 @@
+#ifndef COPCLIB_COPC_EXTENTS_H_
+#define COPCLIB_COPC_EXTENTS_H_
+
+#include <string>
+#include <vector>
+
+#include "copc-lib/las/utils.hpp"
+#include "copc-lib/las/vlr.hpp"
+
+namespace copc
+{
+
+class CopcExtent : public las::CopcExtentsVlr::CopcExtent
+{
+  public:
+    CopcExtent();
+
+    CopcExtent(double minimum, double maximum);
+
+    CopcExtent(const std::vector<double> &vec);
+
+    CopcExtent(const las::CopcExtentsVlr::CopcExtent &other);
+
+    // Convert to lazperf CopcExtent
+    las::CopcExtentsVlr::CopcExtent ToLazPerf() const { return {minimum, maximum}; }
```

Comment:
if we inherit from copcextent, why do we need this method at all?

Reply:
we should either inherit from it and treat it as a lazperf::CopcExtent object, or create our own class that doesn't inherit, and converts to lazperf::CopcExtent